dscreator/datasets/timeseries/__init__.py

```diff
@@ -1,3 +1,3 @@
-from dscreator.datasets.timeseries import msource, sios
+from dscreator.datasets.timeseries import msource, sios, glomma
```

Comment:
Would either change "glomma" to "baterod" or keep "glomma" but change the `BaterodBuilder` to `GlommaBuilder` (in `glomma.py` so maybe let's go with "glomma")

Reply:
are we going with `glomma.py` and `GlommaBuilder` ?

Reply:
Yes, I'd go for that :)